axelrod/strategies/backstabber.py

```diff
@@ -33,3 +33,48 @@ def strategy(self, opponent):
     def reset(self):
         self.D_count = 0
         self.history = []
+
+
+class DoubleCrosser(Player):
+    """
+    Forgives the first 3 defections but on the fourth
+    will defect forever. If the opponent did not defect
+    in the first 6 rounds the player will cooperate until
+    the 180th round. Defects after the 198th round unconditionally.
+    """
+
+    name = 'DoubleCrosser'
+    memory_depth = float('inf')  # Long memory
+
+    def __init__(self):
+        """
+        Initialised the player
+        """
+        super(DoubleCrosser, self).__init__()
+        self.D_count = 0
+        self._initial = 'C'
+        self.stochastic = False
+
+    def strategy(self, opponent):
+        cutoff = 6
+        if len(opponent.history) > 197:
+            return 'D'
+        if not opponent.history:
+            return self._initial
+        if opponent.history[-1] == 'D':
+            self.D_count += 1
+        if len(opponent.history) < 180:
+            if len(opponent.history) > (cutoff):
+                firstelements = []
```

Comment:
this can be written as `firstelements = opponent.history[:cutoff + 1].`

So in fact you could just write `if 'D' not in opponent.history[:cutoff] and opponent.history[-2:] != ['D', 'D']:`

but I'm not sure I understand why you have the `opponent.history[-2:] != ['D', 'D']:` there?